ocaml/xenopsd/scripts/make-custom-xenopsd.conf

```diff
@@ -37,12 +37,12 @@ supported-vbd-backend-kinds=vbd,qdisk,9pfs
 xenguest=${LIBEXECDIR}/xenguest
 
 network-conf=${ETCDIR}/xapi/network.conf
-vif-script=${LIBEXECDIR}/vif
-vif-xl-script=${LIBEXECDIR}/vif
-vbd-script=${LIBEXECDIR}/block
-vbd-xl-script=${LIBEXECDIR}/block
-qemu-vif-script=${LIBEXECDIR}/qemu-vif-script
-setup-vif-rules=${LIBEXECDIR}/setup-vif-rules
+vif-script=${XENOPSD_LIBEXECDIR}/vif
```

Comment:
I would rather stop using the custom `XENOPSD_LIBEXECDIR` and use the standard `LIBEXEC`

Reply:
Could do that.  Maybe @robhoes could give some insight on why that was done initially?

Reply:
Also note, there is a `/usr/libexec/xapi/` hardcoded install location in the Makefile too.

Reply:
Maybe we first fix the discrepancy, and then go for a separate PR to switch to `LIBEXEC`?